webapp/src/app/components/leaderboard/leaderboard.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="flex flex-col gap-4 py-8">
+  <h1 class="text-3xl font-bold">Artemis Leaderboard</h1>
+  @if (query.isPending()) {
+    <span class="text-muted-foreground">Data is loading...</span>
+  } @else if (query.error()) {
+    <span class="text-destructive">An error has occurred</span>
+  }
+
+  @if (leaderboard(); as leaderboard) {
+    <div class="px-4">
+      <app-table>
+        <thead appTableHeader>
+          <tr appTableRow>
+            <th appTableHead>Contributor</th>
+            <th appTableHead>Score</th>
+            <th appTableHead>Activity</th>
+          </tr>
+        </thead>
+        <tbody appTableBody>
+          @for (entry of leaderboard; track entry.githubName) {
+            <tr appTableRow>
+              <td appTableCell>
+                <a href="https://github.com/{{ entry.githubName }}" target="_blank" rel="noopener noreferrer" class="flex items-center gap-2 font-medium">
+                  <img src="{{ entry.avatarUrl }}" width="24" height="24" />
+                  <span class="text-muted-foreground">{{ entry.name }}</span>
+                </a>
+              </td>
+              <td appTableCell>{{ entry.score }}</td>
+              <td appTableCell class="flex items-center gap-4" title="Changes Requested">
+                <div class="flex items-center gap-2">
+                  <app-icon-pull-request-changes-requested />
+                  {{ entry.changesRequested }}
+                </div>
+                <div class="flex items-center gap-2" title="Approvals">
+                  <app-icon-pull-request-approved />
+                  {{ entry.approvals }}
+                </div>
+                <div class="flex items-center gap-2" title="Comments">
+                  <app-icon-pull-request-comment />
+                  {{ entry.comments }}
+                </div>
+              </td>
+            </tr>
+          }
+        </tbody>
+        <tfoot appTableFooter>
```

Comment:
I think we can remove the footer row actually 🤔

Reply:
Absolutely, right now it provides no real value.
Thinking about it, we could also use it to sum up all of the activity in the leaderboard.

Which option would you prefer?

Reply:
I guess that's a neat thing to have but IMO it would only make sense if it also includes some change indicators, see for example here: https://willtrumpwin.com/ 
<img width="97" alt="image" src="https://github.com/user-attachments/assets/1b260bd1-48ca-4ced-ad95-9b566f3e95d5">

For now I would just remove it.